package.json

```diff
@@ -24,8 +24,11 @@
   },
   "dependencies": {
     "@bitwarden/cli": "^1.12.1",
+    "bitwarden-dmenu": "^1.5.9",
     "clipboardy": "^1.2.3",
-    "minimist": "^1.2.5"
+    "minimist": "^1.2.5",
+    "otpauth": "^6.2.3",
+    "otpauth-uri-parser": "^1.0.1"
```

Comment:
I am curious why you want for these packages ([otpauth](https://www.npmjs.com/package/otpauth) and [otpauth-uri-parser](https://www.npmjs.com/package/otpauth-uri-parser)) rather than the more widely used [otplib](https://www.npmjs.com/package/otplib). I know very little about these packages though, so you may know something I dont

Reply:
Honestly, no particular reason. I didn't know about otplib but while skimming through its github page (For some reason its docs are not available rn) you can't pass options such as algorithm and digits while creating the authenticator object, you instead need to set them globally, which I don't want to do. Also otplib doesn't parse otpauth URLs so we would still need otpauth-uri-parser.

Reply:
fair enough. We can add these libs in then